guidelines/index.html

```diff
@@ -372,11 +372,11 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
                   					
                </li>
                					
-               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">The WCAG 2 Documents</a></strong> - A brief introduction to the WCAG 2 supporting documents and supplemental guidance.</p>
+               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/docs/">The WCAG 2 Documents</a></strong> - WCAG 2 の関連文書及び補足的なガイダンスの簡単な紹介。</p>
                   					
                </li>
                
-               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> introduces the new success criteria with persona quotes that illustrate the accessibility issues.</p>
+               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> では、アクセシビリティの問題を説明するペルソナの引用とともに、新しい達成基準を紹介している。</p>
```

Comment:
「ペルソナの引用」は普通に読むと「ペルソナ*を*引用している」と読めると思うのですが、ここでは "persona quotes..." であり「ペルソナ*が*引用している」という文です。もっとも、ここのquoteはいわゆる引用 (どこかに掲載されているものを引用してきている) とは違うように思います。紹介する、引き合いに出す、言及するというニュアンスに近いのではないでしょうか。

動詞のillustrateも訳しにくくて悩ましいのですが、"quote that illustrate X" 全体をざっくりと「Xの例を紹介する」としてしまってはどうでしょうか。……と思ったら introduce があって紹介かぶりになるので、紹介→説明ですかね。実際、リンク先ではペルソナさんが「I can't tell ...」などと発言している体裁で書かれているので、説明でも良いかなと。

たとえばこんな感じ?

```suggestion
               <li><p><strong><a href="https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/">What's New in WCAG 2.2</a></strong> では、アクセシビリティ問題の例を説明するペルソナともに、新しい達成基準を紹介している。</p>
```

Reply:
訳出が難しい文章でしたね・・・
persona quotesはやはり「ペルソナへの言及」であって、ペルソナ*が*言及しているという文章ではないように思いつつ、最終的な翻訳は太田さんの案でしっくりきました。
さすがです。ありがとうございます！